src/com_tjvendors/admin/models/fields/currencylist.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @package     TJVendors
+ * @subpackage  com_tjvendors
+ *
+ * @author      Techjoomla <extensions@techjoomla.com>
+ * @copyright   Copyright (C) 2023 Techjoomla. All rights reserved.
+ * @license     http://www.gnu.org/licenses/gpl-2.0.html GNU/GPL
+ */
+
+// No direct access.
+defined('_JEXEC') or die();
+
+use Joomla\CMS\Form\FormHelper;
+use Joomla\CMS\HTML\HTMLHelper;
+
+jimport('techjoomla.tjmoney.tjmoney');
+
+FormHelper::loadFieldClass('list');
+
+/**
+ * JFormFieldCurrencyList class
+ *
+ * @package     JTicketing
+ * @subpackage  component
+ * @since       1.0
+ */
+
+class JFormFieldCurrencyList extends JFormFieldList
+{
+	/**
+	 * The form field type.
+	 *
+	 * @var   string
+	 * @since 1.6
+	 */
+	protected $type = 'currencylist';
+
+	/**
+	 * Fiedd to decide if options are being loaded externally and from xml
+	 *
+	 * @var   integer
+	 * @since 2.2
+	 */
+	protected $loadExternally = 0;
+
+	/**
+	 * Method to get a list of options for a list input.
+	 *
+	 * @return array An array of JHtml options.
+	 *
+	 * @since   11.4
+	 */
+	protected function getOptions()
+	{
+		$options = array();
+		$currencies = TjMoney::getCurrencies();
```

Comment:
@Sachintechjoomla do we install tj-money library with quick2cart?

if not then this might give errors in Quick2Cart